src-v2/Judge/NTS.Judge.Blazor/Pages/Setup/RfidTags/RfidTagScanner.razor

```diff
@@ -1,41 +1,43 @@
 ﻿@using NTS.Judge.Blazor.Ports
 @using NTS.Domain.Setup.Entities
 
-<MudInputLabel Class="mr-4 mt-2 font-size-12">Scan Rfid Tag</MudInputLabel>
-<MudPaper Class="d-flex flex-row pb-3" Style="height:55px" Elevation="0">
-    @if(@Value!=null && !_awaitingScan){
-        <MudPaper Class="d-flex flex-column align-self-center mt-3">
-            <MudText Class="ml-2 mr-2" Typo="Typo.overline"> @Value.ToString() </MudText>
-        </MudPaper>
-    }
+@inherits BindableValueComponent<Tag>
+
+<MudInputLabel Class="mr-4 mt-2 font-size-12">@Localizer.Get("Scan Rfid Tag")</MudInputLabel>
+<MudStack Style="height:80px" Row="true">
     @if(_awaitingScan)
     {
-        <MudPaper Class="d-flex flex-column mt-2" Elevation="0" >
+        <MudStack Class="mt-3" >
             <MudProgressCircular Class="align-self-center" Color="Color.Primary" Size="Size.Small" Indeterminate="true" />
             <MudText Typo="Typo.overline" Color="Color.Primary">
-                Waiting for Tag Scan
+                @Localizer.Get("Waiting for Tag Scan")
             </MudText>
-        </MudPaper>
-    }
-    else if(Value==null)
-    {
-        <MudButton Class="mt-3" Color="Color.Primary" Size="Size.Small" Variant="Variant.Filled"
-               StartIcon="@Icons.Material.Filled.Add" OnClick="AddTag">Add Tag</MudButton>
+        </MudStack>
     }
     else
     {
-        <MudButton Class="ml-4 mt-3" Color="Color.Error" Size="Size.Small" Variant="Variant.Filled"
-               StartIcon="@Icons.Material.Filled.Delete" OnClick="RemoveTag">Remove Tag</MudButton>
+        if (Value == null)
+        {
+            <MudButton Style="height:32px" Class="mt-3" Color="Color.Primary" Size="Size.Small" Variant="Variant.Filled"
+                       StartIcon="@Icons.Material.Filled.Add" OnClick="AddTag">@Localizer.Get("Add Tag")</MudButton>
+        }
+        else
+        {
+            <MudPaper Style="height:32px" Class="mt-3">
+                <MudText Class="ml-2 mr-2" Typo="Typo.overline"> @Value </MudText>
+            </MudPaper>
+
+            <MudButton Style="height:32px" Class="ml-3 mt-3" Color="Color.Error" Size="Size.Small" Variant="Variant.Filled"
```

Comment:
Why is this height style necessary? We don't want to use custom styles for layouts and colors, because it violates the consistency provided by the Mud theme.